app/dose-event-slider/dose-event-slider.component.ts

```diff
@@ -15,4 +17,26 @@ export class DoseEventSliderComponent {
     @Input()
     public doseEvent: DoseEvent;
```

Comment:
I added a `sliderIcon` variable for updating the icon on the row.  I also set `dose.action` to SKIPPED or TAKEN when the respective button is pressed.  I'm not sure how to set all the variables when the FAB button is pressed though.  Any suggestions?